lib/src/tools/display/display_service_contract.dart

```diff
@@ -0,0 +1,52 @@
+abstract class DisplayService {
+  ///
+  /// Call this method to set brightness for one display
+  ///
+  void setBrightness(DisplayDevice display, double brightness);
+
+  /// Gets whether Display adapter is turn ON.
+  bool get isDarkModeEnabled;
+
+  ///
+  /// Call this method to disable Display adapter
+  ///
+  void disableDarkMode() {
+    this.isDarkModeEnabled = false;
+  }
+
+  ///
+  /// Call this method to enable Display adapter
+  ///
+  void enableDarkMode() {
+    this.isDarkModeEnabled = true;
+  }
+
+  bool get isNightModeEnabled;
+
+  ///
+  /// Call this method to disable Display adapter
+  ///
+  void disableNightMode() {
+    this.isNightModeEnabled = false;
+  }
+
+  ///
+  /// Call this method to enable Display adapter
```

Comment:
Is it proper comment?

Reply:
Changed comment: "Call this method to enable Display Night Mode.".